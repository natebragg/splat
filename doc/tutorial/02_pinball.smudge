pinball
{
   * idle
     [
          coin --> run
     ],
     run
     [
          drain --> idle
     ],
}
