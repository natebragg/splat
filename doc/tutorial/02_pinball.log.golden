pinball[idle]: Unhandled event "drain"
Sending coin event.
Starting runtime.
Current state: idle
Waiting for events to be handled.
Current state: run
Sending drain event.
Current state: run
Waiting for events to be handled.
Current state: idle
